tests/test_resources.py

```diff
@@ -49,5 +49,77 @@ def test_httpverb(self):
             response = method('foo')
             assert response.success
 
+    def test_pagination(self):
+      self.client.url = 'mock://api.com/{param}{?per_page}'
```

Comment:
It's the same for `issues_url` for repos. I pointed that out here: https://github.com/octokit/octokit.py/pull/14#issuecomment-147247048